packages/serverpod_shared/test/password_manager_test.dart

```diff
@@ -176,4 +176,120 @@ development:
       expect(passwords['mySharedPassword'], 'my password');
     });
   });
+
+  group(
+      'Given a reserved env variable when merging custom passwords then an ArgumentError is thrown for env',
+      () {
+    var passwordManager = PasswordManager(runMode: 'development');
+
+    var reservedEnvVariables = [
+      'SERVERPOD_DATABASE_PASSWORD',
+      'SERVERPOD_SERVICE_SECRET',
+      'SERVERPOD_REDIS_PASSWORD',
+    ];
+
+    for (var envName in reservedEnvVariables) {
```

Comment:
Suggestion: Remove the `for` loop and just validate a single case.

I don't think the looping helps the usecase in any way. We might as well remove it so that we get a single reproducable test.

Reply:
Same for other tests.

Reply:
yeah you are probably right